recipe/meta.yaml

```diff
@@ -35,15 +41,19 @@ requirements:
     - click >=8.0,<9.dev0
     - click-plugins >=1.0
     - cligj >=0.5
-    - munch >=2.3.2
+    - importlib-metadata  # [py<310]
+    - six
     - setuptools
     - gdal
     - {{ pin_compatible('numpy') }}
     - shapely  # only option calc we include by default
+  run_constrained:
+    - boto3 >=1.3.1
 
 test:
   source_files:
```

Comment:
the `commands` section is missing

Reply:
They are placed inside `run_test.{bat|sh|py}`